src/lib/debug/debug_bm_service.cpp

```diff
@@ -78,6 +82,47 @@ static bool sys_info_reply_cb(bool ack, uint32_t msg_id, size_t service_strlen,
   return rval;
 }
 
+static bool config_map_callback(bool ack, uint32_t msg_id, size_t service_strlen, const char *service,
+                         size_t reply_len, uint8_t *reply_data) {
+  bool rval = false;
+  printf("Msg id: %" PRIu32 "\n", msg_id);
+  ConfigCborMapSrvReplyMsg::Data reply = {0, 0, 0, 0, NULL};
+  do {
+    if (ack) {
+      if(ConfigCborMapSrvReplyMsg::decode(reply, reply_data, reply_len) != CborNoError) {
+        printf("Failed to decode sys info reply\n");
+        break;
+      }
+      printf("Service: %.*s\n", service_strlen, service);
+      printf("Reply: \n");
+      printf(" * Node id: %" PRIx64 "\n", reply.node_id);
+      printf(" * Partition: %" PRIu32 "\n", reply.partition_id);
+      printf(" * Success: %d\n", reply.success);
+      printf(" * CBOR encoded map len: %" PRIu32 "\n", reply.cbor_encoded_map_len);
+      if(reply.success) {
+        for(uint32_t i = 0; i<reply.cbor_encoded_map_len; i++) {
+          if(!reply.cbor_data) {
+            printf("NULL map\n");
+            break;
+          }
+          printf("%02x ", reply.cbor_data[i]);
+          if(i%16 == 15) {
+            printf("\n");
+          }
```

Comment:
🪄 could you add a comment explaining why 16/15, or a #define for these magical numbers please?